CHANGELOG.md

```diff
@@ -7,11 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [15.0.1] - 2024-01-31
```

Comment:
15.0.1 is a patch release to v15 released after 16.0.0 was out. It shouldn't have been in the changelog to begin with